newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -191,15 +190,20 @@ func readNrqlAlertConditionStruct(condition *newrelic.AlertNrqlCondition, d *sch
 	d.Set("name", condition.Name)
 	d.Set("runbook_url", condition.RunbookURL)
 	d.Set("enabled", condition.Enabled)
-	d.Set("nrql.0.Query", condition.Nrql.Query)
-	d.Set("nrql.0.SinceValue", condition.Nrql.SinceValue)
+	d.Set("type", condition.Type)
+	d.Set("expected_groups", condition.ExpectedGroups)
+	d.Set("ignore_overlap", condition.IgnoreOverlap)
 
 	if condition.ValueFunction == "" {
 		d.Set("value_function", "single_value")
```

Comment:
I know this wasn't your code, but doesn't it look like this can just be replaced with a `Default` field in the schema?

Reply:
Agreed. Let's revisit this after we get this merged :)